src/garage/torch/algos/sac.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         replay_buffer,
         *,  # Everything after this is numbers.
         max_episode_length,
-        max_eval_path_length=None,
+        max_eval_episode_length=None,
```

Comment:
```suggestion
        max_episode_length_eval=None,
```

Reply:
For both of these, please explain what happens if the environment has a `max_episode_length` which is less than these numbers